src/ripple/app/paths/cursor/Liquidity.cpp

```diff
@@ -45,7 +45,7 @@ TER PathCursor::liquidity () const
             << "reverseLiquidity< "
             << "nodeIndex=" << pc.nodeIndex_
             << " resultCode=" << transToken (resultCode)
-            << " transferRate_=" << pc.node().transferRate_
+            << " transferRate_=" << *pc.node().transferRate_
```

Comment:
`boost::optional` asserts, in debug, but dereferencing an uninitialized `optional` in release is UB, and we have seen crashes / segfaults caused by it in the past.